Add Solar-Log integration docs

Ernst79 opened the pull request:
**Description:**
Solar-Log documentation for the new Solar-Log platform

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#27036

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

Ernst79 committed:
Add Solar-Log docs

klaasnicolaas commented on source/_components/solarlog.markdown:
```diff
@@ -0,0 +1,107 @@
+description: "Instructions on how to integrate Solar-Log sensors within Home Assistant."
+logo: solar-log.png
+ha_category: Sensor
+ha_release: 0.100
```
```suggestion
ha_release: "0.100"
```

Ernst79 replied:
Fixed, I also added double quotes around the IoT class.

I've removed the zone.identifier files. I don't know why, but they were created automatically.

klaasnicolaas reviewed (changes requested):
What is the purpose of the zone.identifier files? I think you can leave it out.

Ernst79 committed:
fix suggested changes

Ernst79 committed:
Remove of scan interval configuration option

klaasnicolaas committed:
:pencil2: Tweak

klaasnicolaas committed:
:pencil2: Tweak

klaasnicolaas committed:
:pencil2: Tweak

klaasnicolaas reviewed (dismissed):
✅ Approved. Can be merged as soon as the parent PR gets merged 👍

Happy Hacktoberfest! 🎉

Ernst79 committed:
Update due to component changes

Added the option to use configuration in the user interface.
Removed the requirement to not have http:// in the host address (will be automatically added now if not provided).

Ernst79 commented:
Balloob requested some improvements to the integration, which also had to be added to the docs. Can you review the changes?
- Added the option to use configuration in the user interface.
- Removed the requirement to not have http:// in the host address (will be automatically added now if not provided).

frenck committed:
:pencil2: Tweaks
